frontend/src/app/main/ui.cljs

```diff
@@ -353,6 +353,15 @@
 
        nil)]))
 
+(defn use-prevent-browser-gesture-navigation!
+  []
+  ;; Prevent the browser from interpreting trackpad horizontal swipe as back/forth
+  ;;
+  ;; In theory We could disable this only for the workspace. However gets too unreliable.
+  ;; It is better to be safe and disable for the dashboard as well.
+  (set! (.. js/document -documentElement -style -overscrollBehaviorX) "none")
+  (set! (.. js/document -body -style -overscrollBehaviorX) "none"))
```

Comment:
I think we should name it `prevent-browser-gesture-navigation!`  (removing the `use-` prefix because it is not a react hook and right now use- prefix indicates a react hook)

And move it under `app.util.dom` because there we have all the dom manipulation helpers.